services/core-web/src/components/mine/Permit/PermitConditionExtraction.tsx

```diff
@@ -54,3 +56,29 @@ export const RenderExtractionProgress: FC = () => (
     </Col>
   </Row>
 );
+
+export const RenderExtractionError: FC = () => (
+  <Row align="middle" justify="space-between" gutter={[10, 16]}>
+    <Col span={24}>
+      <Title className="margin-none" level={2}>
+        Permit Conditions
+      </Title>
+    </Col>
+    <Col span={24}>
+      <Row gutter={10} style={{ background: "#fff" }} justify={"center"}>
+        <div className="null-screen fade-in" style={{ maxWidth: "728px" }}>
+          <div>
+            <p className="margin-medium--bottom">
```

Comment:
Not a huge deal, but it might good to use the Typography elements here to make sure any styling specific to that is in place.